packages/search-in-workspace/src/browser/quick-search-in-workspace.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2017 Erisson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import URI from '@theia/core/lib/common/uri';
+import {
+        QuickOpenService,
+        QuickOpenModel,
+        QuickOpenItem,
+        QuickOpenItemOptions
+} from '@theia/core/lib/browser/quick-open/';
+import {
+        injectable,
+        inject
+} from 'inversify';
+import {
+        MenuModelRegistry,
+        MenuContribution,
+        CommandContribution,
+        CommandRegistry,
+        KeybindingContribution,
+        KeybindingRegistry,
+        KeyCode,
+        Key,
+        Modifier,
+        ILogger
+} from '@theia/core';
+import {
+        CommonMenus,
+        WidgetManager,
+        QuickOpenMode,
+        OpenerService,
+        open,
+        Highlight,
+        QuickOpenOptions
+} from '@theia/core/lib/browser';
+import {
+        SearchInWorkspaceService
+} from '../common/search-in-workspace-service';
+import {
+        SearchInWorkspaceResult,
+        SearchInWorkspaceResultCompare,
+        SearchInWorkspaceOptions,
+} from '../common/search-in-workspace-interface';
+import {
+        Range
+} from '@theia/editor/lib/browser';
+import { LabelProvider } from '@theia/core/lib/browser/label-provider';
+
+@injectable()
+export class QuickSearchInWorkspace implements QuickOpenModel {
+        private currentSearchId: number = -1;
```

Comment:
I have not looked into changes yet, but formatting looks strange - 8 spaces for indentation?